### ISSUE TITLE
BlockHeading not displaying introduction

### ISSUE THREAD

**Issue by PeeDeeWhite** [UmbHost/UmbCheckout.StarterKit.Stripe#15]
The BlockHeading.cshtml partial view does not display the introduction if set.

This is because the view is calling HasValue  instead of Value method to display the content

```       
@if (hasIntroduction)
{
        @content.HasValue("introduction");
}
```

**Comment by AaronSadlerUK** [comment]
Hi @PeeDeeWhite,

Many thanks for your PR!

This has been merged and will be published to NuGet soon in version 1.1.5

**Comment by PeeDeeWhite** [comment]
HI Aaron,

You're more than welcome and thank you for a great starter kit and UmbCheckout.

Regards

Paul

________________________________
